dashboard/app/models/script_level.rb

```diff
@@ -587,9 +587,7 @@ def should_hide_survey(user, viewed_user)
   # @return [Hash<String, String>] all information needed to uniquely identify this object across environments.
   def seeding_key(seed_context, use_existing_level_keys = true)
     my_key = {
-      'script_level.level_keys': get_level_keys(seed_context, use_existing_level_keys),
-      'script_level.chapter': chapter,
```

Comment:
Removed these from seeding_key, since we determined that levels, lesson, and script are the only things that should determine uniqueness of a script level.

This will prevent script levels from being recreated when script levels are rearranged / deleted.